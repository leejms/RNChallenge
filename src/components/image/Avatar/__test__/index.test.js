import 'react-native'
import React from 'react'
import Component from '../index'

import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer
    .create(
    <Component
      src='https://www.planwallpaper.com/static/images/9-credit-1.jpg'
      />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})