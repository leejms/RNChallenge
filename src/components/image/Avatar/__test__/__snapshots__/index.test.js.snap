// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "height": 60,
      "justifyContent": "center",
      "width": 60,
    }
  }
>
  <Image
    source={
      Object {
        "uri": "https://www.planwallpaper.com/static/images/9-credit-1.jpg",
      }
    }
    style={
      Object {
        "borderRadius": 30,
        "height": 60,
        "width": 60,
      }
    }
  />
</View>
`;
